src/js/letters/containers/Main.jsx

```diff
@@ -43,6 +41,11 @@ export class Main extends React.Component {
       return awaitingResponse;
     }
 
+    // If address isn't available, take the whole system down
+    if (addressAvailability === unavailable) {
+      return backendServiceError;
```

Comment:
Stepping through `letters.js`, it seems as though we have an action.type that maps to INVALID_ADDRESS_PROPERTY in `constants`. This type doesn't get used anywhere in the app...except in the unit tests where we make sure it works. Might `invalidAddressProperty` be a better one to use in this case? If not, can we just delete it and all its references, which might be even nicer. Thoughts?

Reply:
Yeah, so what I did to test this was use the redux dev tools to dispatch a `GET_ADDRESS_FAILURE`. This made the `addressAvailability` `unavailable`. If we then look at the render function, it will display the system down message if the value returned from this function is `backendServiceError`, so I just made it return that if the address is down. :man_shrugging: 

The `invalidAddressProperty` seems more like it's for when we do get an address, but it's no good. Originally, I think it was used for blank addresses, but now we're just starting in edit mode if we get a blank address.

I think removing it is good. 👍

Reply:
Removed in https://github.com/department-of-veterans-affairs/vets-website/pull/6689